timify-ui/src/components/PodcastGrid/PodcastGrid.jsx

```diff
@@ -3,19 +3,24 @@ import { AiOutlineLoading } from "react-icons/ai"
 import "./PodcastGrid.css"
 import PodcastCard from '../PodcastCard/PodcastCard'
 
-const PodcastGrid = ({ podcasts }) => {
+const PodcastGrid = ({ podcastsByGenre }) => {
 
   return (
-    <>
-        <h3 className='podcast-header'>Podcasts</h3>
-        <div className="podcast-grid">
-            { podcasts.map((podcast, i) => (
-                    <PodcastCard podcast={podcast} key={i} />
-                )
-            )}
+    <div className="podcastGrid">
+      {podcastsByGenre && Object.entries(podcastsByGenre).map(([genre, podcasts]) => (
+        <div key={genre} className="genre-section">
+          <h2>{genre}</h2>
```

Comment:
You may want to create a dict to map each genre param string to a readable string and use it here

Reply:
i was actually wondering how i would achieve that. thank you for this suggestion!